common.lic

```diff
@@ -181,6 +181,14 @@ module DRC
     rummage('B', container)
   end
 
+  def count_boxes(settings)
```

Comment:
If you're up for it, this would be great as a separate PR. Otherwise you're going to have people getting errors and having to restart dependency manually.

Reply:
sure, let me try that